### PR TITLE
Add support for Ubuntu control hosts

### PR DESCRIPTION
**Pull request by markgoddard**

### CODE REVIEW

**Review by oneswig — COMMENTED** [review]
What's the motivation for Ubuntu support for the control host - for Ubuntu laptops acting as control host?  Looking forward to your Mac OS X patches...

The latter patches don't appear connected to Ubuntu portability issues?

**Comment by markgoddard** [comment]
Yes, that's the idea - John and I can then use our laptops as the control host. I would love to add support for Mac OSX, but alas I'd have no means of testing it. Feel free to DIY...

**Comment by markgoddard** [comment]
The latter patches aren't entirely relevant to Ubuntu support - correct, but they are still useful!

File: doc/source/installation.rst [hunk 1]
```diff
@@ -5,26 +5,50 @@ Installation
...
+.. code-block:: console
+   :caption: CentOS
+
+   $ yum install -y git
+
+.. code-block:: console
+   :caption: Ubuntu
 
-    $ yum install -y git
+   $ apt install -y git
```
Comment (JohnGarbutt) [inline doc/source/installation.rst]: I would be tempted to have a section that says, on Ubuntu you do these commands: ... But this works too.

Comment (markgoddard) [reply doc/source/installation.rst]: I've reworked it a little, and simplified the package installation into a single command for each platform. WDYT?

**Pull request merged**